src/extensions/animation/index.js

```diff
@@ -19,9 +19,7 @@ import { useEffect, useRef, useState } from '@wordpress/element';
 const allowedBlocks = [
 	{ animateChildren: false, blockType: 'coblocks/gallery-carousel' },
 	{ animateChildren: true, blockType: 'coblocks/gallery-collage' },
-	{ animateChildren: false, blockType: 'coblocks/gallery-masonry' },
 	{ animateChildren: true, blockType: 'coblocks/gallery-offset' },
-	{ animateChildren: true, blockType: 'coblocks/gallery-stacked' },
```

Comment:
This change needs to be made for the collage block as well correct?

Reply:
yup! Removed.